microprot/scripts/calculate_Neff.py

```diff
@@ -61,10 +61,15 @@ def cluster_sequences(dm, cutoff):
     -------
     list of int
         flat cluster numbers to which sequences are assigned
+
+    Notes
+    -----
+    returns an empty list if the distance matrix is empty (e.g., there is only
+    one input sequence)
     """
     t = 1.0 - cutoff / 100.0
     return list(fcluster(linkage(dm.condensed_form()), t,
```

Comment:
Hi @sjanssen2 thanks for your comments! My consideration for returning an empty list is that the next function in the workflow - `effective_family_size`, can simply take the input without any modification.

I also think that the `x = A if condition else B` thing is a pythonic expression, while Perl only has `do something if condition;`. It can't have `if` and `else` in one line.